index.html

```diff
@@ -0,0 +1,137 @@
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Web Development Summit</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700&family=Righteous&display=swap"
+        rel="stylesheet">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-iYQeCzEYFbKjA/T2uDLTpkwGzCiq6soy8tYaI1GyVh/UjpbCx/TYkiZhlZB6+fzT" crossorigin="anonymous">
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+    <header class="container-fluid">
+        <div class="over-menu d-none d-md-flex justify-content-end py-1 px-4 align-items-center">
+            <i class="fa-brands fa-facebook-f me-3"></i>
+            <i class="fa-brands fa-twitter me-3"></i>
+            <span class="me-3">English</span>
+            <span class="me-3">My Page</span>
+            <span class="me-3">Logout</span>
+        </div>
+        <nav class="main-menu">
+            <a class="navbar-brand logo d-none d-md-block" href="index.html">WebDev'22</a>
+            <div class="menu-icon">
+                <i class="fa-solid fa-bars ms-4 mt-4 mb-4 fa-2x d-md-none"></i>
+            </div>
+            <div class="closeIcon hidden">
+                <i class="fa-solid fa-xmark fa-2xl"></i>
+            </div>
+            <div class="menu-wrapper">
+                <a href="index.html" class="menu-item nav-link nav-item d-md-none">Home</a>
+                <a href="about.html" class="menu-item nav-link nav-item">About</a>
+                <a href="#" class="menu-item nav-item nav-link">Program</a>
+                <a href="#" class="menu-item nav-item nav-link">Join</a>
+                <a href="#" class="menu-item nav-item nav-link">Sponsor</a>
+                <a href="#" class="menu-item nav-item nav-link">News</a>
+                <button type="button" class="menu-button d-none d-md-block">WebDev Campaign</button>
+            </div>
+        </nav>
+        <div class="headline mt-4">
+            <div class="container hl-wrapper">
+                <p>Sharing the knowledge</p>
+                <h1>WebDev Summit 2022</h1>
+                <div class="hl-box">
+                    <p>A celebration to Web Development. 4 days of fast-paced seminars, lectures, exhibitions and many
+                        more activities. Join us in this awesome summit where more than 80 countries come together to share their knowledge on web development</p>
+                </div>
+                <p class="date">2022.12.12(MON) ~ 15(THU)</p>
+                <p class="date-text">@ National Center of New Technologies </p>
+            </div>
+        </div>
+    </header>
+    <main>
+        <section class="main-program pt-4 pb-4">
```

Comment:
- [ ] [REQUIRED] Please fix the issues!
#### 1. Please put a `hover effect` on the program `cards` like [Behance design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) and pay attention to the `height` and `width`!

#### 2. Pay attention to the layout part of the `program section`, because there is unnecessary space, and the layout of the `elements` there is not good!

![image](https://user-images.githubusercontent.com/63915024/190510596-84565af8-8a96-4a2e-8882-4fed73b86939.png)